### ISSUE TITLE
ArgumentError: invalid byte sequence in UTF-8

### ISSUE THREAD

**Issue by Overload119** [ndbroadbent/turbo-sprockets-rails3#108]
Logs

```
/Users/amirsharif/.rvm/rubies/ruby-2.1.3/bin/ruby /Users/amirsharif/.rvm/gems/ruby-2.1.3/bin/rake assets:precompile:all RAILS_ENV=production RAILS_GROUPS=assets --trace
** Invoke assets:precompile:all (first_time)
** Invoke assets:cache:clean (first_time)
** Invoke assets:environment (first_time)
** Execute assets:environment
** Invoke environment (first_time)
** Execute environment
REDIS - [pub-redis-17759.us-east-1-3.6.ec2.redislabs.com:17759] - PING/PONG
** Execute assets:cache:clean
** Execute assets:precompile:all
rake aborted!
ArgumentError: invalid byte sequence in UTF-8
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/turbo-sprockets-rails3-0.3.14/lib/sprockets/static_non_digest_generator.rb:46:in `gsub!'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/turbo-sprockets-rails3-0.3.14/lib/sprockets/static_non_digest_generator.rb:46:in `block in generate'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/sprockets-2.2.3/lib/sprockets/base.rb:219:in `block in each_logical_path'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/sprockets-2.2.3/lib/sprockets/base.rb:206:in `block (2 levels) in each_file'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/sprockets-2.2.3/lib/sprockets/base.rb:196:in `each'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/sprockets-2.2.3/lib/sprockets/base.rb:196:in `each_entry'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/sprockets-2.2.3/lib/sprockets/base.rb:204:in `block in each_file'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/sprockets-2.2.3/lib/sprockets/base.rb:203:in `each'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/sprockets-2.2.3/lib/sprockets/base.rb:203:in `each_file'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/sprockets-2.2.3/lib/sprockets/base.rb:217:in `each_logical_path'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/turbo-sprockets-rails3-0.3.14/lib/sprockets/static_non_digest_generator.rb:29:in `generate'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/turbo-sprockets-rails3-0.3.14/lib/turbo-sprockets/tasks/assets.rake:99:in `internal_precompile'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/turbo-sprockets-rails3-0.3.14/lib/turbo-sprockets/tasks/assets.rake:119:in `block (3 levels) in <top (required)>'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/rake-10.4.2/lib/rake/task.rb:240:in `call'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/rake-10.4.2/lib/rake/task.rb:240:in `block in execute'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/rake-10.4.2/lib/rake/task.rb:235:in `each'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/rake-10.4.2/lib/rake/task.rb:235:in `execute'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/rake-10.4.2/lib/rake/task.rb:179:in `block in invoke_with_call_chain'
/Users/amirsharif/.rvm/rubies/ruby-2.1.3/lib/ruby/2.1.0/monitor.rb:211:in `mon_synchronize'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/rake-10.4.2/lib/rake/task.rb:172:in `invoke_with_call_chain'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/rake-10.4.2/lib/rake/task.rb:165:in `invoke'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/rake-10.4.2/lib/rake/application.rb:150:in `invoke_task'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/rake-10.4.2/lib/rake/application.rb:106:in `block (2 levels) in top_level'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/rake-10.4.2/lib/rake/application.rb:106:in `each'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/rake-10.4.2/lib/rake/application.rb:106:in `block in top_level'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/rake-10.4.2/lib/rake/application.rb:115:in `run_with_threads'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/rake-10.4.2/lib/rake/application.rb:100:in `top_level'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/rake-10.4.2/lib/rake/application.rb:78:in `block in run'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/rake-10.4.2/lib/rake/application.rb:176:in `standard_exception_handling'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/rake-10.4.2/lib/rake/application.rb:75:in `run'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/gems/rake-10.4.2/bin/rake:33:in `<top (required)>'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/bin/rake:23:in `load'
/Users/amirsharif/.rvm/gems/ruby-2.1.3/bin/rake:23:in `<main>'
Tasks: TOP => assets:precompile:all
```

Gemfile.lock

```
GIT
  remote: https://github.com/Filepicker/filepicker-rails
  revision: 18d6215739b29cfa5db052f90b6704b3a88a4d70
  specs:
    filepicker-rails (1.3.0)
      rails (>= 3.2)

GEM
  remote: https://rubygems.org/
  specs:
    Platform (0.4.0)
    actionmailer (3.2.21)
      actionpack (= 3.2.21)
      mail (~> 2.5.4)
    actionpack (3.2.21)
      activemodel (= 3.2.21)
      activesupport (= 3.2.21)
      builder (~> 3.0.0)
      erubis (~> 2.7.0)
      journey (~> 1.0.4)
      rack (~> 1.4.5)
      rack-cache (~> 1.2)
      rack-test (~> 0.6.1)
      sprockets (~> 2.2.1)
    activemodel (3.2.21)
      activesupport (= 3.2.21)
      builder (~> 3.0.0)
    activerecord (3.2.21)
      activemodel (= 3.2.21)
      activesupport (= 3.2.21)
      arel (~> 3.0.2)
      tzinfo (~> 0.3.29)
    activerecord-colored_log_subscriber (0.1.0)
      activerecord (>= 3.2, < 5.0)
    activeresource (3.2.21)
      activemodel (= 3.2.21)
      activesupport (= 3.2.21)
    activesupport (3.2.21)
      i18n (~> 0.6, >= 0.6.4)
      multi_json (~> 1.0)
    addressable (2.3.7)
    arel (3.0.3)
    awesome_print (1.6.1)
    aws-sdk (2.0.31)
      aws-sdk-resources (= 2.0.31)
    aws-sdk-core (2.0.31)
      builder (~> 3.0)
      jmespath (~> 1.0)
      multi_json (~> 1.0)
      multi_xml (~> 0.5)
    aws-sdk-resources (2.0.31)
      aws-sdk-core (= 2.0.31)
    bcrypt (3.1.10)
    better_errors (2.1.1)
      coderay (>= 1.0.0)
      erubis (>= 2.6.6)
      rack (>= 0.9.0)
    binding_of_caller (0.7.2)
      debug_inspector (>= 0.0.1)
    bourbon (4.2.0)
      sass (~> 3.4)
      thor
    builder (3.0.4)
    byebug (3.5.1)
      columnize (~> 0.8)
      debugger-linecache (~> 1.2)
      slop (~> 3.6)
    cancan (1.6.10)
    capistrano (3.2.1)
      i18n
      rake (>= 10.0.0)
      sshkit (~> 1.3)
    capistrano-bundler (1.1.4)
      capistrano (~> 3.1)
      sshkit (~> 1.2)
    capistrano-rails (1.1.2)
      capistrano (~> 3.1)
      capistrano-bundler (~> 1.1)
    capistrano-rails-console (0.5.2)
      capistrano (>= 3.0.0, < 4.0.0)
    capistrano3-unicorn (0.2.1)
      capistrano (~> 3.1, >= 3.1.0)
    capybara (2.4.4)
      mime-types (>= 1.16)
      nokogiri (>= 1.3.3)
      rack (>= 1.0.0)
      rack-test (>= 0.5.4)
      xpath (~> 2.0)
    celluloid (0.16.0)
      timers (~> 4.0.0)
    chewy (0.8.1)
      activesupport (>= 3.2)
      elasticsearch (>= 1.0.0)
    childprocess (0.5.6)
      ffi (~> 1.0, >= 1.0.11)
    chronic (0.10.2)
    coderay (1.1.0)
    coffee-rails (3.2.2)
      coffee-script (>= 2.2.0)
      railties (~> 3.2.0)
    coffee-script (2.3.0)
      coffee-script-source
      execjs
    coffee-script-source (1.8.0)
    colorize (0.7.5)
    columnize (0.9.0)
    connection_pool (2.2.0)
    cookiejar (0.3.2)
    database_cleaner (1.4.0)
    debug_inspector (0.0.2)
    debugger-linecache (1.2.0)
    devise (3.4.1)
      bcrypt (~> 3.0)
      orm_adapter (~> 0.1)
      railties (>= 3.2.6, < 5)
      responders
      thread_safe (~> 0.1)
      warden (~> 1.2.3)
    diff-lcs (1.2.5)
    domain_name (0.5.23)
      unf (>= 0.0.5, < 1.0.0)
    elasticsearch (1.0.12)
      elasticsearch-api (= 1.0.12)
      elasticsearch-transport (= 1.0.12)
    elasticsearch-api (1.0.12)
      multi_json
    elasticsearch-transport (1.0.12)
      faraday
      multi_json
    em-http-request (1.1.2)
      addressable (>= 2.3.4)
      cookiejar
      em-socksify (>= 0.3)
      eventmachine (>= 1.0.3)
      http_parser.rb (>= 0.6.0)
    em-socksify (0.3.0)
      eventmachine (>= 1.0.0.beta.4)
    em-synchrony (1.0.4)
      eventmachine (>= 1.0.0.beta.1)
    em-websocket (0.5.1)
      eventmachine (>= 0.12.9)
      http_parser.rb (~> 0.6.0)
    enum_column3 (0.1.4)
    erubis (2.7.0)
    eventmachine (1.0.7)
    eventmachine_httpserver (0.2.1)
    exception_notification (4.0.1)
      actionmailer (>= 3.0.4)
      activesupport (>= 3.0.4)
    excon (0.44.2)
    execjs (2.5.2)
    factory_girl (4.5.0)
      activesupport (>= 3.0.0)
    fakeredis (0.5.0)
      redis (~> 3.0)
    faraday (0.9.1)
      multipart-post (>= 1.2, < 3)
    fast_stack (0.1.0)
      rake
      rake-compiler
    fastimage (1.6.8)
      addressable (~> 2.3, >= 2.3.5)
    ffi (1.9.6)
    flamegraph (0.1.0)
      fast_stack
    formatador (0.2.5)
    friendly_id (4.0.10.1)
      activerecord (>= 3.0, < 4.0)
    get_process_mem (0.2.0)
    god (0.13.5)
    gon (5.2.3)
      actionpack (>= 2.3.0)
      json
      multi_json
      request_store (>= 1.0.5)
    guard (2.5.1)
      formatador (>= 0.2.4)
      listen (~> 2.6)
      lumberjack (~> 1.0)
      pry (>= 0.9.12)
      thor (>= 0.18.1)
    guard-bundler (2.1.0)
      bundler (~> 1.0)
      guard (~> 2.2)
      guard-compat (~> 1.1)
    guard-compat (1.2.1)
    guard-livereload (2.3.1)
      em-websocket (~> 0.5)
      guard (~> 2.0)
      multi_json (~> 1.8)
    hashie (3.4.0)
    hike (1.2.3)
    hiredis (0.6.0)
    hitimes (1.2.2)
    httmultiparty (0.3.16)
      httparty (>= 0.7.3)
      mimemagic
      multipart-post
    http-cookie (1.0.2)
      domain_name (~> 0.5)
    http_parser.rb (0.6.0)
    httparty (0.13.5)
      json (~> 1.8)
      multi_xml (>= 0.5.2)
    i18n (0.7.0)
    jbuilder (2.2.16)
      activesupport (>= 3.0.0, < 5)
      multi_json (~> 1.2)
    jmespath (1.0.2)
      multi_json (~> 1.0)
    journey (1.0.4)
    jquery-rails (3.1.2)
      railties (>= 3.0, < 5.0)
      thor (>= 0.14, < 2.0)
    json (1.8.3)
    jwt (1.2.1)
    kgio (2.9.3)
    koala (2.0.0)
      addressable
      faraday
      multi_json
    libv8 (3.16.14.7)
    listen (2.8.5)
      celluloid (>= 0.15.2)
      rb-fsevent (>= 0.9.3)
      rb-inotify (>= 0.9)
    lmdb (0.4.8)
    local_time (1.0.2)
      coffee-rails
    loggability (0.11.0)
    lumberjack (1.0.9)
    mail (2.5.4)
      mime-types (~> 1.16)
      treetop (~> 1.4.8)
    mailchimp-api (2.0.6)
      excon (>= 0.16.0)
      json (>= 1.7.7)
    mechanize (2.7.2)
      domain_name (~> 0.5, >= 0.5.1)
      http-cookie (~> 1.0.0)
      mime-types (~> 1.17, >= 1.17.2)
      net-http-digest_auth (~> 1.1, >= 1.1.1)
      net-http-persistent (~> 2.5, >= 2.5.2)
      nokogiri (~> 1.4)
      ntlm-http (~> 0.1, >= 0.1.1)
      webrobots (>= 0.0.9, < 0.2)
    memory_profiler (0.0.4)
    method_source (0.8.2)
    mime-types (1.25.1)
    mimemagic (0.2.1)
    mini_portile (0.6.2)
    mono_logger (1.1.0)
    msgpack (0.5.11)
    multi_json (1.10.1)
    multi_xml (0.5.5)
    multipart-post (2.0.0)
    mysql2 (0.3.18)
    net-http-digest_auth (1.4)
    net-http-persistent (2.9.4)
    net-scp (1.2.1)
      net-ssh (>= 2.6.5)
    net-ssh (2.9.2)
    newrelic_rpm (3.10.0.279)
    nokogiri (1.6.6.2)
      mini_portile (~> 0.6.0)
    ntlm-http (0.1.1)
    oauth (0.4.7)
    oauth2 (1.0.0)
      faraday (>= 0.8, < 0.10)
      jwt (~> 1.0)
      multi_json (~> 1.3)
      multi_xml (~> 0.5)
      rack (~> 1.2)
    omniauth (1.2.2)
      hashie (>= 1.2, < 4)
      rack (~> 1.0)
    omniauth-facebook (2.0.0)
      omniauth-oauth2 (~> 1.2)
    omniauth-oauth2 (1.2.0)
      faraday (>= 0.8, < 0.10)
      multi_json (~> 1.3)
      oauth2 (~> 1.0)
      omniauth (~> 1.2)
    open4 (1.3.4)
    orm_adapter (0.5.0)
    polyglot (0.3.5)
    popen4 (0.1.2)
      Platform (>= 0.4.0)
      open4 (>= 0.4.0)
    pry (0.10.1)
      coderay (~> 1.1.0)
      method_source (~> 0.8.1)
      slop (~> 3.4)
    puffing-billy (0.5.1)
      addressable
      em-http-request (~> 1.1.0)
      em-synchrony
      eventmachine
      eventmachine_httpserver
      http_parser.rb (~> 0.6.0)
      multi_json
    quiet_assets (1.1.0)
      railties (>= 3.1, < 5.0)
    rack (1.4.7)
    rack-attack (4.2.0)
      rack
    rack-cache (1.2)
      rack (>= 0.4)
    rack-livereload (0.3.15)
      rack
    rack-mini-profiler (0.9.2)
      rack (>= 1.1.3)
    rack-protection (1.5.3)
      rack
    rack-ssl (1.3.4)
      rack
    rack-test (0.6.3)
      rack (>= 1.0)
    rails (3.2.21)
      actionmailer (= 3.2.21)
      actionpack (= 3.2.21)
      activerecord (= 3.2.21)
      activeresource (= 3.2.21)
      activesupport (= 3.2.21)
      bundler (~> 1.0)
      railties (= 3.2.21)
    rails-dev-tweaks (1.2.0)
      actionpack (>= 3.1)
      railties (>= 3.1)
    rails-secrets (1.0.2)
      rails (>= 3.2.0, <= 4.1.0)
    rails_autolink (1.1.6)
      rails (> 3.1)
    rails_param (0.0.3)
    railties (3.2.21)
      actionpack (= 3.2.21)
      activesupport (= 3.2.21)
      rack-ssl (~> 1.3.2)
      rake (>= 0.8.7)
      rdoc (~> 3.4)
      thor (>= 0.14.6, < 2.0)
    raindrops (0.13.0)
    rake (10.4.2)
    rake-compiler (0.9.5)
      rake
    rb-fsevent (0.9.4)
    rb-inotify (0.9.5)
      ffi (>= 0.5.0)
    rdoc (3.12.2)
      json (~> 1.4)
    react-rails (1.0.0)
      coffee-script-source (~> 1.8)
      connection_pool
      execjs
      rails (>= 3.1)
      react-source (~> 0.13)
    react-source (0.13.2)
    redcarpet (3.2.2)
    redd (0.7.5)
      faraday (~> 0.9.1)
      fastimage (~> 1.6.6)
      hashie (~> 3.4.0)
      multi_json (~> 1.10.1)
    redis (3.2.1)
    redis-actionpack (3.2.4)
      actionpack (~> 3.2.0)
      redis-rack (~> 1.4.4)
      redis-store (~> 1.1.4)
    redis-activesupport (3.2.5)
      activesupport (~> 3.2.0)
      redis-store (~> 1.1.0)
    redis-namespace (1.5.1)
      redis (~> 3.0, >= 3.0.4)
    redis-rack (1.4.4)
      rack (~> 1.4.0)
      redis-store (~> 1.1.4)
    redis-rails (3.2.4)
      redis-actionpack (~> 3.2.4)
      redis-activesupport (~> 3.2.4)
      redis-store (~> 1.1.4)
    redis-store (1.1.4)
      redis (>= 2.2)
    ref (1.0.5)
    request_store (1.1.0)
    responders (1.1.2)
      railties (>= 3.2, < 4.2)
    resque (1.25.2)
      mono_logger (~> 1.0)
      multi_json (~> 1.0)
      redis-namespace (~> 1.3)
      sinatra (>= 0.9.2)
      vegas (~> 0.1.2)
    resque-scheduler (4.0.0)
      mono_logger (~> 1.0)
      redis (~> 3.0)
      resque (~> 1.25)
      rufus-scheduler (~> 3.0)
    resque_spec (0.16.0)
      resque (>= 1.19.0)
      rspec-core (>= 3.0.0)
      rspec-expectations (>= 3.0.0)
      rspec-mocks (>= 3.0.0)
    roman-numerals (0.3.0)
    rspec-core (3.2.0)
      rspec-support (~> 3.2.0)
    rspec-expectations (3.2.0)
      diff-lcs (>= 1.2.0, < 2.0)
      rspec-support (~> 3.2.0)
    rspec-mocks (3.2.0)
      diff-lcs (>= 1.2.0, < 2.0)
      rspec-support (~> 3.2.0)
    rspec-rails (3.2.0)
      actionpack (>= 3.0, <= 4.2)
      activesupport (>= 3.0, <= 4.2)
      railties (>= 3.0, <= 4.2)
      rspec-core (~> 3.2.0)
      rspec-expectations (~> 3.2.0)
      rspec-mocks (~> 3.2.0)
      rspec-support (~> 3.2.0)
    rspec-support (3.2.1)
    ruby-prof (0.15.4)
    rubyzip (1.1.7)
    rufus-scheduler (3.0.9)
      tzinfo
    rvm1-capistrano3 (1.3.2.2)
      capistrano (~> 3.0)
      sshkit (>= 1.2)
    sass (3.4.13)
    sass-rails (3.2.6)
      railties (~> 3.2.0)
      sass (>= 3.1.10)
      tilt (~> 1.3)
    selenium-webdriver (2.46.2)
      childprocess (~> 0.5)
      multi_json (~> 1.0)
      rubyzip (~> 1.0)
      websocket (~> 1.0)
    sequel (3.48.0)
    simple_oauth (0.3.1)
    sinatra (1.4.5)
      rack (~> 1.4)
      rack-protection (~> 1.4)
      tilt (~> 1.3, >= 1.3.4)
    slim (3.0.2)
      temple (~> 0.7.3)
      tilt (>= 1.3.3, < 2.1)
    slop (3.6.0)
    soundcloud (0.3.1)
      hashie
      httmultiparty (>= 0.3)
      httparty (>= 0.7.3)
    spring (1.3.6)
    spring-commands-rspec (1.0.4)
      spring (>= 0.9.1)
    sprockets (2.2.3)
      hike (~> 1.2)
      multi_json (~> 1.0)
      rack (~> 1.0)
      tilt (~> 1.1, != 1.3.0)
    sqlite3 (1.3.10)
    sshkit (1.7.1)
      colorize (>= 0.7.0)
      net-scp (>= 1.1.2)
      net-ssh (>= 2.8.0)
    stackprof (0.2.7)
    temple (0.7.5)
    therubyracer (0.12.1)
      libv8 (~> 3.16.14.0)
      ref
    thor (0.19.1)
    thread_safe (0.3.4)
    tilt (1.4.1)
    timers (4.0.1)
      hitimes
    treetop (1.4.15)
      polyglot
      polyglot (>= 0.3.1)
    turbo-sprockets-rails3 (0.3.14)
      railties (> 3.2.8, < 4.0.0)
      sprockets (>= 2.2.0)
    tzinfo (0.3.44)
    uglifier (2.7.0)
      execjs (>= 0.3.0)
      json (>= 1.8.0)
    unf (0.1.4)
      unf_ext
    unf_ext (0.0.6)
    unicorn (4.8.3)
      kgio (~> 2.6)
      rack
      raindrops (~> 0.7)
    unicorn-rails (2.2.0)
      rack
      unicorn
    unicorn-worker-killer (0.4.3)
      get_process_mem (~> 0)
      unicorn (~> 4)
    validate_url (1.0.0)
      activemodel (>= 3.0.0)
      addressable
    vegas (0.1.11)
      rack (>= 1.0.0)
    warden (1.2.3)
      rack (>= 1.0)
    webrobots (0.1.1)
    websocket (1.2.2)
    whenever (0.9.4)
      chronic (>= 0.6.3)
    will_paginate (3.0.7)
    wisper (1.6.0)
    wisper-rspec (0.0.2)
    wordnet (1.0.0)
      loggability (~> 0.5)
      sequel (~> 3.38)
    xpath (2.0.0)
      nokogiri (~> 1.3)
    youtube_it (2.4.2)
      builder
      excon
      faraday (>= 0.8, < 0.10)
      json (~> 1.8)
      nokogiri (~> 1.6.0)
      oauth (~> 0.4.4)
      oauth2 (~> 1.0.0)
      simple_oauth (>= 0.1.5)
    yt (0.22.0)
      activesupport

PLATFORMS
  ruby

DEPENDENCIES
  activerecord-colored_log_subscriber
  awesome_print
  aws-sdk (~> 2.0)
  better_errors
  binding_of_caller
  bourbon
  byebug
  cancan
  capistrano (~> 3.2.1)
  capistrano-bundler
  capistrano-rails
  capistrano-rails-console
  capistrano3-unicorn
  capybara
  chewy
  coffee-rails
  coffee-script-source (~> 1.8.0)
  database_cleaner
  devise
  elasticsearch
  enum_column3
  exception_notification (~> 4.0.1)
  execjs
  factory_girl
  fakeredis
  filepicker-rails!
  flamegraph
  friendly_id (~> 4.0.10)
  god
  gon
  guard (~> 2.5.1)
  guard-bundler
  guard-livereload
  hashie
  hiredis
  jbuilder
  jquery-rails
  koala
  lmdb
  local_time
  mailchimp-api
  mechanize
  memory_profiler
  msgpack
  mysql2
  newrelic_rpm
  nokogiri
  omniauth
  omniauth-facebook
  popen4
  puffing-billy
  quiet_assets
  rack-attack
  rack-livereload
  rack-mini-profiler
  rails (= 3.2.21)
  rails-dev-tweaks
  rails-secrets
  rails_autolink
  rails_param
  react-rails
  react-source (= 0.13.2)
  redcarpet
  redd
  redis
  redis-rails
  resque
  resque-scheduler
  resque_spec
  roman-numerals
  rspec-rails
  ruby-prof
  rvm1-capistrano3
  sass-rails
  selenium-webdriver
  slim
  soundcloud
  spring
  spring-commands-rspec
  sqlite3
  stackprof
  therubyracer
  turbo-sprockets-rails3
  uglifier
  unicorn
  unicorn-rails
  unicorn-worker-killer
  validate_url
  whenever
  will_paginate
  wisper
  wisper-rspec
  wordnet
  youtube_it
  yt

```

**Comment by Overload119** [comment]
This was working fine yesterday, and the Gemfile.lock hasn't been touched.
It could be an asset change, but still doesn't make sense to me.

**Comment by Overload119** [comment]
Okay, so I added 

```
require "byebug"
byebug
```

To the conflicting line in the Gem, and it appears a demon has done something terrible to the asset file. Still investigating...

![screen shot 2015-09-09 at 5 03 49 pm](https://cloud.githubusercontent.com/assets/1012072/9774187/da6866a4-5714-11e5-885a-9d9aba2932cd.png)

**Comment by Overload119** [comment]
Unfortunately this is some transient error.

The culprit was a file that has been unchanged for years in the `vendors/`
The file (probably not important) was code for Chosen.JS.

I added a one-line comment to the file, "// testing" then tried recompiling and it worked.
Then I removed what I added, saved, and compiled and it worked again. 

So I essentially didn't do anything but resave the file (with no net changes) and the assets compiled again.
